docs/source/tutorials/8_inverse_kinematics.py

```diff
@@ -0,0 +1,90 @@
+### Adapted from: https://github.com/kevinzakka/mink/examples
+# REQUIRES:
+# Python 3.9
+# MINK -- pip install mink[examples]
+
+import os
+
+import mujoco
+import mujoco.viewer
+import numpy as np
+from loop_rate_limiters import RateLimiter
+
+import mink
+
+os.chdir("../../../myosuite/simhive/myo_sim/arm")
+_XML_ARM_Model = "myoarm.xml"
+
+xml_string = f"""
+        <mujoco model="MyoArm with Mocap">
+            <include file="{_XML_ARM_Model}"/>
+            <worldbody>
+                <body name="target" pos="0 0 0" quat="0 1 0 0" mocap="true">
+                    <geom type="box" size=".15 .15 .15" contype="0" conaffinity="0" rgba=".6 .3 .3 .2"/>
+                </body>
+            </worldbody>
+        </mujoco>
+        """
+
+model = mujoco.MjModel.from_xml_string(xml_string)
+data = mujoco.MjData(model)
+
+## =================== ##
+## Setup IK.
+## =================== ##
+
+configuration = mink.Configuration(model)
+
+tasks = [
+    end_effector_task := mink.FrameTask(
+        frame_name="S_grasp",
+        frame_type="site",
+        position_cost=1.0,
+        orientation_cost=1.0,
+        lm_damping=1.0,
+    ),
+    posture_task := mink.PostureTask(model=model, cost=1e-2),
+]
+
+## =================== ##
+
+# IK settings.
+solver = "quadprog"
+pos_threshold = 1e-4
+ori_threshold = 1e-4
+max_iters = 20
+
+with mujoco.viewer.launch_passive(
```

Comment:
maybe use the renderer class, so that this can be rendered both onscreen and offscreen.

Reply:
for this examample we support only on-line rendering